corehq/apps/custom_data_fields/models.py

```diff
@@ -54,10 +54,8 @@ def _is_match(field):
         return filter(_is_match, self.fields)
 
     @classmethod
-    def get_or_create(cls, domain, field_type):
-        # todo: this overrides get_or_create from DocumentBase but with a completely different signature.
-        # This method should probably be renamed.
-        existing = cls.view(
+    def get_by_domain_and_type(cls, domain, field_type):
+        return cls.view(
```

Comment:
while you were in here maybe take the opportunity to move to the `dbaccessors` module?